moveit2_tutorials.repos

```diff
@@ -2,7 +2,7 @@ repositories:
   moveit2:
     type: git
     url: https://github.com/ros-planning/moveit2
-    version: 2.3.2
+    version: galactic
```

Comment:
Do we wanna use a tag, or the latest galactic?

Reply:
I'm ok with using the branch in this case.  I was the one who pushed for the tag for more stability for the tutorials but I'm not sure it was helpful because we only push into these branches when we are releasing and we want the tutorials to at least stay up to date with our releases.